src/include/migraphx/operators.hpp

```diff
@@ -1356,6 +1356,108 @@ struct undefined
     argument compute(const shape&, const std::vector<argument>&) const { return {{}, nullptr}; }
 };
 
+struct batch_contiguous
+{
+    int axis = 0;
+    std::vector<int> slice_dims;
+    std::string name() const { return "batch_contiguous"; }
+
+    shape compute_shape(std::vector<shape> inputs) const
+    {
```

Comment:
Add a check_shapes() call (similar to the other operators).

Reply:
will do.